backend/src/main/java/gov/cdc/usds/simplereport/db/repository/PatientPreferencesRepository.java

```diff
@@ -0,0 +1,9 @@
+package gov.cdc.usds.simplereport.db.repository;
+
+import gov.cdc.usds.simplereport.db.model.PatientPreferences;
+import java.util.Optional;
+import java.util.UUID;
+
+public interface PatientPreferencesRepository extends AuditedEntityRepository<PatientPreferences> {
+  Optional<PatientPreferences> findByPersonInternalId(UUID personId);
```

Comment:
I suspect you always have the Person object here, so possibly just using `findByPerson` would be better?

Reply:
In the QueueMutationResolver's addPatientToQueue and updateTimeOfTestQuestions we _only_ have a patientId, and I didn't want to unnecessarily add a getPatient call there. Otherwise, yes, related code could just use `findByPerson`. I'll add the additional signature to the interface and remove the couple of unnecessary `getInternalId` calls, though